### PR TITLE
Svcat help improvements

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

File: cmd/svcat/class/get_cmd.go [hunk 1]
```diff
@@ -39,14 +39,14 @@ func (c *getCmd) SetFormat(format string) {
 func NewGetCmd(cxt *command.Context) *cobra.Command {
 	getCmd := &getCmd{Context: cxt}
 	cmd := &cobra.Command{
-		Use:     "classes [name]",
+		Use:     "classes NAME",
```
Comment (carolynvs) [inline cmd/svcat/class/get_cmd.go]: Same as above about optional positional arguments.

File: cmd/svcat/command/command.go [hunk 2]
```diff
@@ -142,3 +143,26 @@ func determineOutputFormat(flags *pflag.FlagSet) (string, error) {
 		return "", fmt.Errorf("invalid --output format %q, allowed values are table, json and yaml", format)
 	}
 }
+
+// NormalizeExamples removes leading and trailing empty lines
+// from the command's Example string and normalizes the indentation
+// so that all examples across all commands are indented consistently.
+func NormalizeExamples(examples string) string {
+	// TODO: this code copied from a pending PR to k8s.io/kubernetes/pkg/kubectl/cmd/templates/normalizers.go; replace this with a call to that method when PR is merged
```
Comment (carolynvs) [inline cmd/svcat/command/command.go]: Would you please link to the PR in the comment?

File: cmd/svcat/instance/get_cmd.go [hunk 3]
```diff
@@ -36,15 +36,15 @@ func (c *getCmd) SetFormat(format string) {
 func NewGetCmd(cxt *command.Context) *cobra.Command {
 	getCmd := &getCmd{Namespaced: command.NewNamespacedCommand(cxt)}
 	cmd := &cobra.Command{
-		Use:     "instances [name]",
+		Use:     "instances NAME",
```
Comment (carolynvs) [inline cmd/svcat/instance/get_cmd.go]: Same as above about optional positional arguments.

File: cmd/svcat/plan/get_cmd.go [hunk 4]
```diff
@@ -47,10 +47,10 @@ func (c *getCmd) SetFormat(format string) {
 func NewGetCmd(cxt *command.Context) *cobra.Command {
 	getCmd := &getCmd{Context: cxt}
 	cmd := &cobra.Command{
-		Use:     "plans [name]",
+		Use:     "plans NAME",
```
Comment (carolynvs) [inline cmd/svcat/plan/get_cmd.go]: Same as above about optional positional arguments.

File: cmd/svcat/plugin/manifest.go [hunk 5]
```diff
@@ -70,6 +70,9 @@ type Plugin struct {
 	// Name of the command for the help text. Required.
 	Name string `yaml:"name"`
 
+	// Use is the one-line description of how the command is used.
+	Use string `yaml:"use"`
```
Comment (carolynvs) [inline cmd/svcat/plugin/manifest.go]: This is not a supported field in the kubectl plugin format and should not be added.

https://github.com/kubernetes/kubernetes/blob/e85b81bbee098a7ec75cc894a8785867ec586798/pkg/kubectl/plugins/plugins.go#L47-L55

File: cmd/svcat/binding/get_cmd.go [hunk 6]
```diff
@@ -36,15 +36,15 @@ func (c *getCmd) SetFormat(format string) {
 func NewGetCmd(cxt *command.Context) *cobra.Command {
 	getCmd := &getCmd{Namespaced: command.NewNamespacedCommand(cxt)}
 	cmd := &cobra.Command{
-		Use:     "bindings [name]",
+		Use:     "bindings NAME",
```
[the remark this answers is not in the document]
Comment (luksa) [reply cmd/svcat/binding/get_cmd.go]: Oops. That's what happens when you code on auto-pilot. Started with the sync command, where NAME is required; then did the same in all other files, without realizing the name obviously isn't required.

File: cmd/svcat/plugin/manifest.go (hunk 5 above)
Comment (carolynvs) [reply cmd/svcat/plugin/manifest.go]: Ah thanks, that's why I needed a link to the PR. I didn't realize that Use was added in it. 👍

**Pull request merged**